node/api/server/server.go

```diff
@@ -76,9 +83,18 @@ func New(APIaddr string, requiredUserAgent string, requiredPassword string, node
 		},
 		done:              make(chan struct{}),
 		listener:          listener,
+		node:              node,
 		requiredUserAgent: requiredUserAgent,
 	}
 
+	// fund the node
+	for i := types.BlockHeight(0); i <= types.MaturityDelay; i++ {
```

Comment:
same here, funding a node before returning it is meant to be something you do when creating a `siatest` server/client pair, not something you do when you create a production server